src/lib/Scenes/Artwork/Components/CommercialButtons/__tests__/CommercialButtons-tests.tsx

```diff
@@ -43,76 +34,78 @@ const componentWithQuery = async ({ mockArtworkData, mockOrderMutationResults, m
   })
 }
 
+const relayComponent = async ({ artwork }) => {
+  return await renderRelayTree({
+    Component: CommercialButtonsFragmentContainer,
+    query: graphql`
+      query CommercialButtonsTestsRenderQuery {
+        artwork(id: "artworkID") {
+          ...CommercialButtons_artwork
+        }
+      }
+    `,
+    mockData: { artwork },
+  })
+}
+
 describe("CommercialButtons", () => {
-  it("renders button for Contact Gallery button if isInquireable", () => {
+  it("renders button for Contact Gallery button if isInquireable", async () => {
     const artwork = {
-      ...defaultArtwork,
+      ...ArtworkFixture,
       isAcquireable: false,
       isOfferable: false,
       isInquireable: true,
     }
-    const component = mount(
-      <CommercialButtons
-        relay={{ environment: {} } as RelayProp}
-        artwork={artwork as CommercialButtonProps["artwork"]}
-      />
-    )
-    expect(component.text()).toContain("Contact gallery")
+    const commercialButtons = await relayComponent({
+      artwork,
+    })
+    expect(commercialButtons.text()).toContain("Contact gallery")
   })
 
-  it("renders Make Offer button if isOfferable", () => {
+  it("renders Make Offer button if isOfferable", async () => {
     const artwork = {
-      ...defaultArtwork,
+      ...ArtworkFixture,
       isAcquireable: false,
       isOfferable: false,
       isInquireable: true,
     }
-    const component = mount(
-      <CommercialButtons
-        relay={{ environment: {} } as RelayProp}
-        artwork={artwork as CommercialButtonProps["artwork"]}
-      />
-    )
-    expect(component.text()).toContain("Contact gallery")
+    const commercialButtons = await relayComponent({
+      artwork,
+    })
+    expect(commercialButtons.text()).toContain("Contact gallery")
```

Comment:
This test case appears to not match the description. looks like the description is correct but the test case is a duplication of the one above.

Reply:
Fixed this.